benchmarks/dynamo/pr_time_benchmarks/benchmark_runner.sh

```diff
@@ -18,8 +18,16 @@ output_file=$1
 # Set the directory of Python programs
 python_programs_dir=$2
 # Loop through all files in the directory of Python programs
+
+start=`date +%s`
+
 for file in $python_programs_dir/*.py
 do
     # Execute the Python program and append the output to the output file
    python $file $output_file
 done
+end=`date +%s`
+
+runtime=$((end-start))
+echo "total time to run benchmarks is:"
+echo $runtime
```

Comment:
Why don't you just put this in the oss signpost (the one that I wrote about in my fbscribelogger post)? Then we'll have historical data too

Reply:
let me find that post